policy/policy.go

```diff
@@ -17,56 +17,88 @@ const (
 // e.g. automated, or locked.
 type Policy string
 
-func Parse(s string) Policy {
-	for _, p := range []Policy{
-		Locked,
-		Automated,
-		Ignore,
-	} {
-		if s == string(p) {
-			return p
-		}
+func Boolean(policy Policy) bool {
+	switch policy {
+	case Locked, Automated, Ignore:
+		return true
 	}
-	return None
+	return false
 }
 
 type Updates map[flux.ServiceID]Update
 
 type Update struct {
```

Comment:
Changing this type also changes the effective argument type in the API, which breaks the backwards-compatible routes (which, to be fair, were put in place long after this PR was started). It is not a huge deal since we will shortly be removing them; just noting FTR.